Makefile

```diff
@@ -52,31 +52,15 @@ docker:
 
 .PHONY: docker-all
 docker-all: docker
-	$(DOCKER_BUILDX) --platform linux/amd64 -t $(REGISTRY)/oauth2-proxy:latest-amd64 .
 	$(DOCKER_BUILDX_X_PLATFORM) -f Dockerfile -t $(REGISTRY)/oauth2-proxy:${VERSION} .
-	$(DOCKER_BUILDX) --platform linux/amd64 -t $(REGISTRY)/oauth2-proxy:${VERSION}-amd64 .
```

Comment:
Ahh, I think there was some confusion over what I meant there, I had meant that we should leave the existing tags but not extend the list of tags, so the new arch would only be available with the not suffixed tags, but the existing ones would be available on both.

Is that too confusing from an end user perspective do you think?

Reply:
Why would we want to have that duplicated? Just to prevent a breaking change? I would like to move away from platform tagged images but if you think that is too much change for the next release I can add these back.
I think we should at least remove all mention of these tagged releases as nobody should be using them anymore.

Reply:
I want to avoid a breaking change for now, we have a bunch of breaking changes in the pipeline, so I'd rather we keep pushing the old tags for now and then drop these when we merge a few more breaking changes

We should definitely remove mentions from the docs and mention the tags are deprecated